src/host/time_updater.h

```diff
@@ -11,24 +11,25 @@ namespace asynchost
 {
   class TimeUpdaterImpl
   {
-    using TClock = std::chrono::steady_clock;
-    TClock::time_point creation_time;
+    using TClock = std::chrono::system_clock;
```

Comment:
This is changing from a monotonic clock to one that is not. We need to carefully check that all logic that derives elapsed deltas (eg. consensus timeouts) is correctly clamping negative values to zero, or expose this in addition to the monotonic clock used for timeout purposes.

Reply:
The only accessor to this within the enclave, `enclave::get_enclave_time()`, already does this clamping to ensure the observed time never moves backwards.